subprojects/docs/src/docs/release/notes.md

```diff
@@ -47,6 +47,14 @@ For example:
 
 In some terminals, this link will be clickable and take you directly to the documentation.
 
+## Gradle source code for plugin developers in IDEs
+
+Plugin authors using Eclipse will now have sources of `gradleApi()`, `gradleTestKit()` and `localGroovy()` dependencies downloaded and attached for navigation in the IDE.
+
+In IDEA, the sources of `gradleApi()` only are attached when [Gradle wrapper](https://docs.gradle.org/6.2/userguide/gradle_wrapper.html#sec:adding_wrapper) is used with `-all` distribution. 
```

Comment:
I fixed this when I merged it, but links in the release notes to other docs should be relative. 

So instead of `https://docs.gradle.org/6.2/userguide/gradle_wrapper.html#sec:adding_wrapper`, you'd just use `userguide/gradle_wrapper.html#sec:adding_wrapper`.  This way the links work even in nightly builds.